titus_isolate/monitor/cgroup_metrics_provider.py

```diff
@@ -67,7 +67,7 @@ def __get_usage_path(self, resource_key) -> Union[str, None]:
             log.warning("No '{}' path for workload: '{}'".format(resource_key, self.__workload.get_id()))
             return None
 
-        if not os.path.isfile(usage_path):
```

Comment:
was this intentionally changed or did it sneak in?

Reply:
Intentional.  The variable wasn't being set as it should be.